src/bin/cli.ts

```diff
@@ -19,6 +19,7 @@ program
     'For every package.json found, generate a CODEOWNERS entry using the maintainers field',
     false
   )
+  .option('--group-source-comments', 'Group rules by source, reducing the amount of comments added', false)
```

Comment:
@gustavkj I can use some help with the `help` command. Is this a  meaningful description to you?

lemme know what else can be added

Reply:
I like what @cyamonide suggested for the readme, but maybe a bit shorter.

> `Instead of generating one comment per rule, enabling this flag will group them, reducing comments to one per source file. Useful if your codeowners file gets too noisy.`

Here are a few alternatives: 
- `Instead of generating one comment per rule, enabling this flag will group them, reducing comments to one per source file.`
- `Reduce the number of generated comments to one per source file`.
- `Reduce the number of generated comments to one per source file instead of one per line`. 
- `Generate one comment per source file`

Reply:
Picked the first one, so we have a bit of everyone in the sentence 👥